source/includes/_disputes.md

```diff
@@ -1160,6 +1160,69 @@ You will also need to attach the Braintree transaction id using the `charge` par
 
 You can always reconnect your Braintree account from the settings page [here](/dashboard/settings/processors) to grant Chargehound access to the disputes API, this will make your integration easier.
 
+## Stripe listing needs response
+
+> Example usage:
+
+```sh
+curl https://api.chargehound.com/v1/disputes?needs_response&warning_needs_response \
```

Comment:
these parameters are not correct. should be `state=needs_response&state=warning_needs_response` I believe